src/service/gameService.js

```diff
@@ -318,7 +320,43 @@ const startNextRound = game => {
     return game
 }
 
+async function save_newroom(game) {
+    try {
+    const room = await Room.create({
+        "players": game.players, 
+        "maxPlayers": game.maxPlayers, 
+        "name": game.name, 
+        "hand": game.hand, 
+        "maxBuyIn": game.maxBuyIn, 
+        "bigBlind": game.bigBlind,  
+        "smallBlind": game.smallBlind, 
+        "bets": game.bets, 
+        "lastToRaiseId": game.lastToRaiseId, 
+        "pot": game.pot, 
+        "playersWaiting": game.playersWaiting, 
+        "phase": game.phase, 
+        "deck": game.deck,
+        "communityCards": game.communityCards,
+        "allInHands": game.allInHands,
+        "sidePots": game.sidePots,
+        "winners": game.winners,
+        "endedByFold": game.endedByFold,
+        "numBots": game.numBots,
+        "moveHistory": game.moveHistory,
+    })
+    console.log("room saved")
```

Comment:
Check indentation. You would like to run `npm test` from the root folder to make sure that your code meets the linting requirements of the project.

Reply:
done, used "Format document"